apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -817,7 +817,7 @@ def append_var(
     Writes new rows to an existing ``var`` dataframe. (This is distinct from ``update_var``
     which mutates the entirety of the ``var`` dataframe, e.g. to add/remove columns.)
 
-    Example:
+    Example::
```

Comment:
Sphinx grossly misrenders this unless we have the double colon. Slightly uglier to have a double colon for help(foo) in Python; much uglier to not have the double colon for Sphix.